### DISCUSSION TITLE
How to refresh specific filament form input? (Select)

### DISCUSSION THREAD

**Discussion by DanjBethel** [filamentphp/filament#13731] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.93

### How can we help you?

Seems like a pretty simple thing, I'm probably just missing something. But after using the "createOptionUsing" method on a Select, how do I refresh its options? The new option isn't being showed automatically for me.

I am also not using relationships for this particular select box, I'm just showing some options from a collection.

**Comment by dissto** [comment, marked as answer]
Can you show some code? 
You probably didn't return the newly created model id from the [`createOptionUsing`](https://filamentphp.com/docs/3.x/forms/fields/select#customizing-new-option-creation) method 🤔

**Reply by DanjBethel** [reply]
Ah, I was returning the entire model and not the ID. Thanks. Figured it was something simple
